mmv1/products/apigee/terraform.yaml

```diff
@@ -359,6 +359,50 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       test_env_vars:
         org_id: :ORG_ID
         billing_account: :BILLING_ACCT
+  EnvKeystore: !ruby/object:Overrides::Terraform::ResourceOverride
+    autogen_async: true
+    import_format: ["{{env_id}}/keystores/{{name}}", "{{env_id}}/{{name}}"]
+    delete_url: '{{env_id}}/keystores/{{name}}'
+    skip_sweeper: true
+    examples:
+    - !ruby/object:Provider::Terraform::Examples
+      # This is a more verbose version of the above that creates all
+      # the resources needed for the acceptance test.
+      name: "apigee_environment_keystore_test"
+      primary_resource_id: "apigee_environment_keystore"
+      test_env_vars:
+        org_id: :ORG_ID
+        billing_account: :BILLING_ACCT
+      skip_docs: true
+      # Resource creation race
+      skip_vcr: true
+    timeouts: !ruby/object:Api::Timeouts
+      insert_minutes: 30
+      delete_minutes: 30
+    custom_code: !ruby/object:Provider::Terraform::CustomCode
+      custom_import: templates/terraform/custom_import/apigee_environment_keystore.go.erb
+  EnvReferences: !ruby/object:Overrides::Terraform::ResourceOverride
+    autogen_async: true
+    import_format: ["{{env_id}}/references/{{name}}", "{{env_id}}/{{name}}"]
+    delete_url: '{{env_id}}/references/{{name}}'
```

Comment:
Is this delete_url different from the normal one that will be implied by the resource's base_url?

Reply:
delete_url is the same base_url and name